pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -1583,14 +1583,14 @@
             <item row="1" column="0">
              <widget class="QCheckBox" name="osdShowFPS">
               <property name="text">
-               <string>Show FPS</string>
+               <string>Show Internal FPS</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="osdShowMessages">
               <property name="text">
-               <string>Show Notifications</string>
+               <string>Show OSD Messages</string>
```

Comment:
Aren't they all technically OSD Messages? Not just subtype of Notifications.

Reply:
I could copy "Show Notifications" to the other duplicate. Your call here.